internal/web/image.go

```diff
@@ -0,0 +1,21 @@
+package web
+
+import (
+	"os"
+	"path/filepath"
+)
+
+func prepareDockerfile(tempdir string) error {
+	// Copy the Dockerfile
+	dockerFile, err := assets.ReadFile("assets/Dockerfile")
```

Comment:
we are reading to write it? Cant we have then the file directly here as pure text? and just write it? Or is it used somewhere else?

Reply:
yup good idea, ended up doing it that way with a template